PassportOfficeOpenAPI.yaml

```diff
@@ -291,46 +376,36 @@ paths:
               $ref: '#/components/schemas/Person'
       responses:
         '200':
-          description: OK, accepted
-        '409':
-          description: Data conflict
-        '401':
-          description: Authorisation required
-        '452': #Custom error
-          description: Bad Data
+          description: OK, entity updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Person'
+        '400':
+          description: Bad Request - invalid syntax
+        '422':
+          $ref: '#/components/responses/InvalidData'
     get:
       tags:
         - Person
       summary: get persons by full name
       operationId: getPersonsByName
       parameters:
-        - name: name
-          in: query
-          description: The name of the person
-          required: true
-          schema:
-            type: string
-        - name: surname
+        - name: ID
           in: query
-          description: The surname of the person
+          description: The ID of the person
           required: true
           schema:
             type: string
-        - name: patronymic
-          in: query
-          description: The patronymic of the person
-          required: false
-          schema:
-            type: string
       responses:
         '200':
-          description: OK, accepted
+          description: OK
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Person'
-        '401':
-          description: Authorisation required
+                type: array
+                items:
+                  $ref: '#/components/schemas/Person'
```

Comment:
It's worth having a /person/{id} resource for getting a single person, and keep `/person` for getting a collection